src/interactivity/Interactivity.js

```diff
@@ -167,13 +171,39 @@ export default class Interactivity {
         });
     }
 
+    _subscribeToLayerEvents (layers) {
+        layers.forEach(layer => {
+            layer.on('updated', () => {
+                this._onLayerUpdated(layer.isAnimated());
```

Comment:
Instead of doing this and the debouncing in one way or another I would create a promise here (checking first that the promise doesn't exist to call `on_mouseMove` just once per frame. Something like:
```
layer.on('updated', ()=>{
    if (!this._layerUpdateTrigger){
        this._layerUpdateTrigger = new Promise(resolve => {
              this._onMouseMove(this._mouseEvent);
              this._layerUpdateTrigger = null;
        });
   }
});

Reply:
I think this can be implemented just with a flag instead of a Promise. However, I'm a bit worried about the number of events fired when there are lots of animated layers. With the debounce, we can filter that amount of calls and improve performance.